spec/features/saml/saml_spec.rb

```diff
@@ -34,17 +34,11 @@
       sp.save
     end
 
-    it 'returns the user to the account page after authentication' do
```

Comment:
As part of the saml_idp update, I updated a conditional in the `valid?` method that was checking if the acs_url value was `nil?` -- because it could evaluate to an empty string, I updated the check to `blank?`

This caused this test to fail. I talked it over with a couple folks and it seems like the correct behavior here should be a failure -- SAML SPs need to have an acs_url registered, and redirecting the user to the account page feels like it's hiding a bug. 

Would love to hear feedback about this behavior, or if we'd rather have it do something else!

Reply:
seems reasonable to show an error like that to me 👍

Reply:
Is it possible to rescue the error and replicate the current behavior for now, and enter a bug ticket to update the behavior? I'm also good with merging as-is, but it's not my call to make!